EvidenceApi/DocumentTypes.json

```diff
@@ -272,5 +272,31 @@
         "description": "A valid letter from the occupant of the property stating that the parent and the child are currently residing with them."
       }
     ]
+  },
+  {
+    "name": "Housing Management",
+    "id": "10",
+    "documentTypes": [
+      {
+        "id": "proof-of-id",
+        "title": "Proof of Identity (ID)",
+        "description": "We will need 2 types of Proof of Identity for the proposed joint tenant. You need to submit two forms of ID. The below are documents you can provide: Current Passport, Full photo Driver's Licence, Photo Provisional driving licence, Freedom Pass, Full Birth Certificate, Immigration documents from Home Office confirming then status, Valid resident's Permit (EU nationals), NHS Medical Card, Divorce Letter from the court, Identity Card, Marriage Certificate, National Insurance, any other form of ID. Please note: For passport data page with photo is required. For driver's license/residence permit/identity card, front and back is required. Any government document must be in goverment offical header"
```

Comment:
I remember in the meeting with the MMH team Bella showed them how it would look like on the page and they agreed. I also tried adding "\n" but that is not picked up.